src/apps/reservation-list/list/reservation-list.tsx

```diff
@@ -45,273 +40,99 @@ const ReservationList: FC<ReservationListProps> = ({ pageSize }) => {
   const { open } = useModalButtonHandler();
   const { pauseReservation, deleteReservation, reservationDetails } =
     getModalIds();
-  const [reservation, setReservation] = useState<ReservationType | null>(null);
+  const [reservationWithDetails, setReservationWithDetails] =
+    useState<ReservationType | null>(null);
   const [reservationToDelete, setReservationToDelete] =
     useState<ReservationType | null>(null);
   const { data: userData } = usePatronData();
-  const [modalDetailsId, setModalDetailsId] = useState<string | null>(null);
-  const [modalDeleteId, setModalDeleteId] = useState<string | null>(null);
-  // Data fetch
-  const {
-    isSuccess: isSuccessFBS,
-    data,
-    isLoading: isLoadingFBS
-  } = useGetReservationsV2();
 
   const {
-    isSuccess: isSuccessPublizon,
-    data: publizonData,
-    isLoading: isLoadingPublizon
-  } = useGetV1UserReservations();
-
-  // State
-  const [readyForPickupReservationsFBS, setReadyForPickupReservationsFBS] =
-    useState<ReservationType[] | ReservationDetailsV2[] | null>(null);
```

Comment:
All this deletion is a result of using `useReservations()` and business objects leading to much fewer states and effects.